descriptors/ModuleDescriptor-template.json

```diff
@@ -296,6 +296,11 @@
           "pathPattern": "/data-import/uploadUrl/subsequent",
           "permissionsRequired": ["data-import.uploadUrl.get"]
         },
+        {
+          "methods": ["GET"],
```

Comment:
Could you please update the 'methods' property and the permission (as data-import.downloadUrl.post) with POST  to reflect the method of this endpoint? Also pls remove the redundant '/' in the pathPattern.

Reply:
My bad, it should've been `GET` in the first place.  Fixed the path and updated the original endpoint to be `GET`.